sidebars.js

```diff
@@ -113,6 +113,8 @@ module.exports = {
                 label: 'Semgrep in continuous integration (CI)',
                 items: [
                     'semgrep-ci/overview',
+                    'semgrep-ci/manual-ci-setup',
+                    'semgrep-ci/semgrep-ci-with-app',
```

Comment:
I suggest ordering the “with app” page before the “manual CI setup” page.

Reply:
This is done!